tools/wpt/browser.py

```diff
@@ -1282,6 +1282,11 @@ def install_webdriver(self, dest=None, channel=None, browser_binary=None):
 
         version = self.version(browser_binary)
         if version is None:
+            # Check if the user has given a Chromium binary.
+            chromium = Chromium(self.logger)
+            if chromium.version(browser_binary):
+                raise ValueError("Provided binary is a Chromium binary and should be run using "
+                                 "Chromium commands, e.g. \"./wpt run chromium\"")
```

Comment:
I would say that "run" is the (sub)command and "chromium" is an argument. To sidestep that:

```suggestion
                                 "\"./wpt run chromium\" or similar.")
```